data/raw.dvc

```diff
@@ -1,4 +1,6 @@
 outs:
+- md5: a532c2e43fdb5903eecfa5d4f1601582.dir
+  size: 438914978
```

Comment:
It looks like this was merged with "accept both" which is not what we want here. Merging dvc files is difficult and I haven't written instructions for that yet. So in the meantime I recommend going through the dvc resources and Github merging in https://github.com/nasaharvest/crop-mask/wiki/2.-Getting-Started

Reply:
Hmmm, alrightyy, I'll take a look. Thanks